src/main.zig

```diff
@@ -29,6 +29,9 @@ pub fn main() void {
         bun.win32.STDOUT_FD = bun.toFD(std.io.getStdOut().handle);
         bun.win32.STDERR_FD = bun.toFD(std.io.getStdErr().handle);
         bun.win32.STDIN_FD = bun.toFD(std.io.getStdIn().handle);
+
+        // This fixes printing unicode characters
+        _ = std.os.windows.kernel32.SetConsoleOutputCP(65001);
```

Comment:
is there a constant in the standard library we can use?

Reply:
<img width="626" alt="image" src="https://github.com/oven-sh/bun/assets/24465214/8a1d3520-21b0-4fe8-bd5b-1427ab406742">
only in c++ :(